tests/imputers/test_BaseImputer.py

```diff
@@ -2,202 +2,268 @@
 
 import numpy as np
 import pandas as pd
+import polars as pl
 import pytest
-import test_aide as ta
 from sklearn.exceptions import NotFittedError
 
 import tests.test_data as d
+from tests import utils as u
 from tests.base_tests import (
     ColumnStrListInitTests,
     GenericFitTests,
     OtherBaseBehaviourTests,
 )
 
 
+@pytest.fixture
+def library(request):
+    return request.param
+
+
+@pytest.fixture()
```

Comment:
Ah we're getting some errors with e.g. MeanImputer tests because the tests can't see the fixtures - since we already have some data creation functions in this script, I would just convert these fixtures into functions for now which should get round this